pywatershed/version.py

```diff
@@ -1,3 +1,12 @@
-# pywatershed version file automatically created using update_version.py on July 12, 2023 10:58:55
+# pywatershed version file automatically created using update_version.py on July 18, 2023 14:36:23
 
-__version__ = "0.1.2"
```

Comment:
@w-bonelli the author stuff is not needed on main?

Reply:
I assumed it should be on both main and develop. It hasn't been on develop since https://github.com/EC-USGS/pywatershed/commit/c82b98a528b33b6e81cf29a8234b6bf13c612e85#diff-1478550bd0058c77f80b63f84458953b950ab22911fe879ac703ecca6162b639